System.IO.Packaging/FileFormatException.cs

```diff
@@ -8,12 +8,10 @@
 //
 //---------------------------------------------------------------------------
 
-using System;
-using System.Security;
-using System.IO.Packaging;
-
 namespace System.IO
```

Comment:
I don't think it is a good idea to declare a namespace outside of `System.IO.Packaging` or any other namespace within that (e.g., `System.IO.Packaging.Properties`).